.taskcluster.yml

```diff
@@ -24,9 +24,15 @@ tasks:
           go get -t github.com/taskcluster/taskcluster-cli/... 
           && cd  /go/src/github.com/taskcluster/taskcluster-cli 
           && git fetch {{ event.head.repo.url }} {{ event.head.ref }} 
-          && git checkout {{ event.head.sha }} 
           && make 
-          && go test ./...
+          && go get -u github.com/alecthomas/gometalinter 
+          && gometalinter --install
+          && go get -u ./...
+          && git checkout {{ event.head.sha }} 
```

Comment:
These lines have me a little worried -- it's running `go get -u` on the default branch (the one `go get -t github.com/taskcluster/taskcluster-cli/...` found), then switching to the commit we want to test.  If those have different dependent packages, would this still work?

@walac this is pretty far over my golang head -- can you comment here?